Gemfile

```diff
@@ -38,7 +34,7 @@ gem 'annotate', '~> 3.0'
 # Use Sidekiq to enqueue job in the background
 gem 'sidekiq', '~> 6.0.2'
 
-gem 'activeadmin', '~> 2.6.0'
+gem 'activeadmin', '~> 2.7'
```

Comment:
Looking https://activeadmin.info/0-installation.html section `Upgrading` seems that you might need to update these files:
app/admin/dashboard.rb
config/initializers/active_admin.rb